<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>541</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Loma</family>
    <pointsize>12</pointsize>
    <weight>75</weight>
    <italic>true</italic>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Abyssinica SIL</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>430</y>
      <width>521</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Abyssinica SIL</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Ustaw parametry plików</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Abyssinica SIL</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Log</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="iplineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>125</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Abyssinica SIL</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Abyssinica SIL</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>IP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="downloadpushButton">
    <property name="geometry">
     <rect>
      <x>309</x>
      <y>10</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Abyssinica SIL</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Pobierz</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="portlineEdit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Abyssinica SIL</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>22</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>401</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Abyssinica SIL</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="numberOfFiles">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Abyssinica SIL</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="log">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>571</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Abyssinica SIL</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>150</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Abyssinica SIL</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Wyczysc</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>120</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Abyssinica SIL</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Otwórz Docelowy</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Komenda</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>380</y>
      <width>441</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>find /home/radek/Wideo/ -maxdepth 1 -type f -printf &quot;%f\n&quot;</string>
    </property>
   </widget>
   <widget class="QLabel" name="STATUS">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>341</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>587</width>
     <height>32</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlik">
    <property name="title">
     <string>Plik</string>
    </property>
    <addaction name="actionWyj_cie"/>
   </widget>
   <addaction name="menuPlik"/>
  </widget>
  <action name="actionWyj_cie">
   <property name="text">
    <string>Wyjście</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
